include/fields2cover/utils/random.h

```diff
@@ -52,6 +53,13 @@ class Random {
   /// @return Double \f$\in [min, max]\f$
   double getRandomExp(double min, double max);
 
+  /// Get a random double in a range [0;+inf) with exponential
+  /// probability.
+  /// Lower values are more probable than higher ones.
+  /// @param lambda is the rate parameter
+  /// @return Double \f$\in [min, max]\f$
+  double getRandomExp(double lambda);
```

Comment:
getRandomExp do different things when has only one parameter than when it has two. One could expect the single parameter version to return a value between (0, lambda) which is not the case. Could you put another name to this function? Like getRandomExpDist(double lambda)

Reply:
Yes. I will change the method name.